src/sentry/runner/commands/run.py

```diff
@@ -599,6 +599,7 @@ def occurrences_ingest_consumer(**options):
 @click.option("--output-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
 @click.option("--ingest-profile", required=True)
 @click.option("--indexer-db", default="postgres")
+@click.option("--join-timeout", type=int, help="Join timeout in seconds.", default=5)
```

Comment:
if my other comment is right, could we make the default something a little more permissive? something like 60-120s?